README.md

```diff
@@ -0,0 +1,154 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">React.js</a></li>
+    <li><a href="">Redux</a></li>
+    <li><a href="">HTML/CSS/a></li>
+  </ul>
+  <summary>API</summary>
+  <ul>
+    <li><a href="">[Rest Countries](https://restcountries.com/#rest-countries)</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+
+- Browse a comprehensive list of all countries
+- Quickly search for countries by name
+- View detailed information on each country
+- Up-to-date data powered by REST Countries API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://worldindata.onrender.com/)
+- [Project Documentation](https://www.loom.com/share/6de321c38de2425b9716f4f8dbba6c67)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Install
+
+Install this project with:
+1. Installed Git.
+2. Create a local directory that you want to clone the repository.
+3. Open the command prompt in the created directory.
+4. On the terminal run this command git clone <https://github.com/Hisoka37/Worldindata.git>
+5. Run ``` npm i --save-dev ```  to Install the dev dependencies
+### Usage
+
+To run the project, execute the following command:
+``` npm run build ```
+
+### Run tests
+
+To run tests, run the following command:
+``` npm test ```
+
+### Deployment
+
+You can deploy this project using:
+``` npm run deploy ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Walid Kabboussa**
+
+- GitHub: [@Hisoka37](https://github.com/Hisoka37)
+- Twitter: [@KBwalid9](https://twitter.com/KBwalid9)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/walidkb/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Make it Faster and Add other data fields.**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to open a Github issues.
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly send to me an e-mail expressing it, it would make my day and fuel my motivation.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse team for this journey.
+
```

Comment:
- [ ] hi, you have done really well with the project, :clap: , Just a few things to fix and it should be completed, first please notice that it is mentioned in the project requirements **The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.** So in your acknowledgments section  please provide the appropriate credit to **Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).**